### ISSUE TITLE
RC Test: Bending Light 1.1.0-rc.1

### ISSUE THREAD

**Issue by samreid** [phetsims/tasks#682]
Bending Light is ready for an RC test.  We would like to republish it to fix the "WebGL wave view not captured in screenshots" issue https://github.com/phetsims/bending-light/issues/350 present in the current version on the website.  This version is built from master (not a maintenance release), hence we will need to make sure changes to common code have not disturbed anything.  The issue that prepared this RC is https://github.com/phetsims/bending-light/issues/351

**Link to Sim**
http://www.colorado.edu/physics/phet/dev/html/bending-light/1.1.0-rc.1/bending-light_en.html

**Link to iFrame**
http://www.colorado.edu/physics/phet/dev/html/bending-light/1.1.0-rc.1/bending-light_en-iframe.html

**Link to Test Matrix**
https://docs.google.com/spreadsheets/d/1__9A1CBoW9C5xrWVfADcR1zR09PhuKbIJC-b275LMKM/edit#gid=671507842

For each issue you report, please identify whether the same issue exists in the version currently published on the PhET website.

No new strings have been added--I'll leave it up to @ariel-phet if he wants to run any `?stringTest` tests.

Edit: Since this is being done off master would be good to do the full normal battery of tests, to make sure all is well

**Please also verify**
- [x] stringTest=double
- [x] stringTest=long
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (should not redirect, OK to look bad, test on one desktop platform)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test

**Comment by ariel-phet** [comment]
@phet-steele definitely no need to be done until #679 is complete (GAO is much higher priority to get published)

**Comment by ariel-phet** [comment]
Moving to low priority for the moment considering other current testing tasks

**Comment by phet-steele** [comment]
@samreid, ready to deploy after changes in phetsims/bending-light/issues/362.

**Comment by samreid** [comment]
1.1.0 was published here https://phet.colorado.edu/sims/html/bending-light/latest/bending-light_en.html

**Comment by samreid** [comment]
Translations check out, SHAs are not newer, so won't need to be updated in sim_deployment.md.  I updated third-party-licenses.md.  I'll email the team.

**Comment by samreid** [comment]
I emailed the team, closing.
